plugins/web/active/Nikto_Unauthenticated@OWTF-WVS-002.py

```diff
@@ -9,11 +9,13 @@
 
 
 def run(PluginInfo):
-    # ServiceLocator.get_component("config").Show()
     plugin_helper = ServiceLocator.get_component("plugin_helper")
     resource = ServiceLocator.get_component("resource")
-    NiktoOutput = plugin_helper.CommandDump('Test Command', 'Output', resource.GetResources('Nikto_Unauth'), PluginInfo, [])
-    Content = plugin_helper.CommandDump('Test Command', 'Output', resource.GetResources('Nikto_Verify_Unauth'), PluginInfo, NiktoOutput)
+    NiktoOutput = plugin_helper.CommandDump('Test Command', 'Output', resource.GetResources('Nikto_Unauth'),
+                                            PluginInfo, [])
+    Content = plugin_helper.CommandDump('Test Command',
```

Comment:
Maintain consistency with 

```
plugin.CommandDump(xxx
    yyyy,
    zzzz,
)

@depierre @7a we can use other other convention because sometimes the parameters args are so long - they have to broken into multiple lines - affecting readability.